FPEngine.cc

```diff
@@ -149,6 +149,17 @@ extern "C" int g_has_npcap_loopback;
  * network resources and schedule packet transmissions. */
 FPNetworkControl global_netctl;
 
+/* To send decoy probe one global data structure of type map has been used.
+ * It stores packet which has unique packet id which is concatenation of 
+ * ProbeID + decoynumber + target_host's number*/
+static std::map<std::string, IPv6Header*> decoy_probes;
+
+std::string get_unique_id(const char *id, int decoynumber, const char *targetIP) {
+  std::string id_str(id), ip_str(targetIP);
+  std::stringstream out;
+  out<<id_str<<decoynumber<<ip_str;
```

Comment:
We can use any character as delimiter right ? I mean something like `out<<id_str<<","<<decoynumber<<","<<ip_str;` let me know if I am missing something.